dataset/create_dataset.py

```diff
@@ -91,15 +96,18 @@ def create_dataset(
     parser = argparse.ArgumentParser(description="Generate dataset")
     parser.add_argument("-t", "--text_path", help="Path to text file", type=str, required=True)
     parser.add_argument("-a", "--audio_path", help="Path to audio file", type=str, required=True)
-    parser.add_argument(
-        "-f", "--forced_alignment_path", help="Path to forced alignment JSON", type=str, default="align.json"
-    )
     parser.add_argument("-o", "--output_path", help="Path to save snippets", type=str, default="wavs")
-    parser.add_argument("-u", "--unlabelled_path", help="Path to save unlabelled clips", type=str, default="unlabelled")
-    parser.add_argument(
-        "-l", "--label_path", help="Path to save snippet labelling text file", type=str, default="metadata.csv"
-    )
-    parser.add_argument("-i", "--info_path", help="Path to save info file", type=str, default="info.json")
+    parser.add_argument("-lang", "--language", help="The language to use", type=str, default="English")
```

Comment:
Remove `-lang`. This should be a single letter (or not there at all)

Reply:
done